mod-users-bl/src/main/resources/domain/mod-users-bl/features/configurePermissions.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Configure permissions for admin
+
+  Background:
+    * url baseUrl
+    * call login testAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-tenant': #(testTenant), 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+
+    * table blUserPermissions
+      | name                                |
+      | 'users-bl.item.get'                 |
+      | 'users-bl.transactions.get'         |
+      | 'users-bl.item.delete'              |
+
+    * def testAdminUserId = "00000000-1111-5555-9999-999999999991"
+
+  Scenario: Add bl-users permissions to admin user
```

Comment:
Why is adding permissions to a user a test scenario?

Reply:
before adding any module's permission, module should be enabled first.
for example:-
* table modules
      | name                                |
      | 'okapi'                             |
      | 'mod-permissions'                   |
      | 'mod-configuration'                 |
      | 'mod-users'                         |
      | 'mod-login'                         |

and **mod-users-bl** cannot be enabled before mod-authtoken(mod-authtoken is enabled as the last step of setup-users.feature).  So it is not possible to add permissions in **users-bl-junit.feature** file directly . Thats why created a separate feature file (configurepermissions.feature ) and adding permissions from there to the user.

Reply:
That's another unfortunate trade off of the design of these tests. Please add some comments to explain why this feature is needed to work around those design choices so future maintainers can understand this.

Reply:
Has documentation been added to explain why this is needed? I couldn't find any in the changes since this comment.

Reply:
yes , comments has been added in "configurepermission.feature" & users-bl.feature file.

Reply:
The comments in this file seem to explain what is happening, not the reason why it is necessary: 
`
mod-users-bl cannot be enabled before mod-authtoken(mod-authtoken is enabled as the last step of setup-users.feature). So it is not possible to add permissions in users-bl-junit.feature file directly
`

Reply:
Added below lines 
# /bl-users/login requires a token to be created.
  # In reality mod-login should depend on mod-authtoken.. But for bootstrap reasons, it doesn't (only an optional dependency). https://issues.folio.org/browse/MODLOGIN-155

Reply:
Added below comments....

/bl-users/login requires a token to be created.
In reality mod-login should depend on mod-authtoken.. But for bootstrap reasons, it doesn't (only an optional dependency). https://issues.folio.org/browse/MODLOGIN-155

Reply:
Ok, thank you